### DISCUSSION TITLE
Is it possible to use mantine components without using the theming system?

### DISCUSSION THREAD

**Discussion by trishagonzales** [mantinedev/mantine#5817] [Q&A] [answered]
I have design tokens and patterns written in plain css and referenced in tailwind config via custom properties. If I migrate to mantine, is it also necessary to use the theming system for components to work?

Or can I use my tokens directly in each component without configuring them in a theme object? I mainly use css modules for styling and a bit of tailwind.

Thanks for this awesome library btw.

**Comment by rtivital** [comment, marked as answer]
If you do not need Mantine theming features, you can use your own styling library.
https://mantine.dev/styles/css-modules/#styling-mantine-components-without-css-modules

**Reply by trishagonzales** [reply]
thanks
